packages/init/src/index.ts

```diff
@@ -49,8 +52,14 @@ async function init() {
   // install the required dependencies in the project.
   await installRequiredDependencies(cwd, frameworkResult.value);
 
+  const sliceLibPath = await (() => {
```

Comment:
We can change this notation by 
 
`
const sliceLibPath = lib ? await installLib(cwd, lib, branch) : null
`

The default case is explicit and we avoid the IIFE overload

Reply:
it's undefined not null here but yeah if you prefer